packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.controller.js

```diff
@@ -0,0 +1,57 @@
+import moment from 'moment';
+import { DATA_PROCESSING_API_STATUSES } from '../../data-processing.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $state,
+    $timeout,
+    $uibModal,
+    CucCloudMessage,
+    dataProcessingJobLogsService,
+  ) {
+    this.$scope = $scope; // router state
```

Comment:
```suggestion
    this.$scope = $scope; 
```
Is this used ? There is no need to inject it if it's not the case

Reply:
$scope is used but not following $state, i'll remove it